Base/res/ladybird/inspector.js

```diff
@@ -192,6 +192,89 @@ inspector.addAttributeToDOMNodeID = nodeID => {
     pendingEditDOMNode = null;
 };
 
+inspector.setStyleSheets = styleSheets => {
+    const styleSheetPicker = document.getElementById("style-sheet-picker");
+    const styleSheetSource = document.getElementById("style-sheet-source");
+    styleSheetPicker.replaceChildren();
+    styleSheetSource.innerHTML = "";
+
+    function addOption(styleSheet, text) {
+        const option = document.createElement("option");
+        option.innerText = text;
+        if (styleSheet.type) option.dataset["type"] = styleSheet.type;
```

Comment:
I have a big dislike of inlined if statements, can add braces so `prettier` keeps them on multiple lines? (here and in other places below)